src/data_watcher.cpp

```diff
@@ -191,6 +191,10 @@ std::optional<DataOperation>
          */
         return processCreate(receivedEventInfo);
     }
+    else if ((std::get<EventMask>(receivedEventInfo) & IN_MOVED_TO) != 0)
```

Comment:
Commit Message:

Please include why this event is required to process in the commit message.